neo4j/test-integration/dbconn_test.go

```diff
@@ -65,7 +65,7 @@ func makeRawConnection(logger log.Logger) (dbserver.DbServer, db.Connection) {
 }
 
 func BenchmarkQuery(b *testing.B) {
-	_, conn := makeRawConnection(&log.Console{Debugs: true, Errors: true, Infos: true, Warns: true})
+	_, conn := makeRawConnection(&log.Console{Debugs: false, Errors: true, Infos: true, Warns: true})
```

Comment:
either that or I introduce some kind of log id-based filtering into the logger config. WDYT @2hdddg?

Reply:
Not sure what the question is. The benchmarks doesn't run anywhere as far as I know

Reply:
it runs in CI and with the new messages being logged in DEBUG, the tests would time out.

Reply:
(notably the `RECORD` message)

Reply:
Are you sure you run the benchmarks in CI? You shouldn't, it might be that they are but that is a mistake

Reply:
In any case, they revealed a real issue: when pulling many records, (debug) logging can become a bottleneck. What do I do about this?

Reply:
As discussed offline, merging this for now and creating another PR to move the Bolt logging to a separate utility